src/code/photo_album.asm

```diff
@@ -231,33 +232,36 @@ func_028_4176::
     ret                                           ;; 28:4184 $C9
 
 
-func_028_4185::
+; Loads in the "OK" tiles for the unlocked photos
+PhotoAlbumImageHandler::
+    ; If Debug3 is set, set all photos enabled
     ld   a, [ROM_DebugTool3]                      ;; 28:4185 $FA $05 $00
     and  a                                        ;; 28:4188 $A7
-    jr   z, .else_4195_28                         ;; 28:4189 $28 $0A
+    jr   z, .notDebug                             ;; 28:4189 $28 $0A
 
     ld   a, $FF                                   ;; 28:418B $3E $FF
     ld   [wPhotos1], a                            ;; 28:418D $EA $0C $DC
     ld   a, $0F                                   ;; 28:4190 $3E $0F
     ld   [wPhotos2], a                            ;; 28:4192 $EA $0D $DC
-.else_4195_28:
+.notDebug:
     ld   bc, $01                                  ;; 28:4195 $01 $01 $00
 .loop_4198_28
```

Comment:
I wasn't sure if there was a convention for renaming these loops or not. It's just a basic for loop.

Reply:
Not really, and in general any kind of label is better than the generic `jr_` ones